packages/core/src/cookie.ts

```diff
@@ -57,7 +57,7 @@ export function getCookie(name: string) {
   return findCommaSeparatedValue(document.cookie, name)
 }
 
-export function areCookiesAuthorized(useSecureCookie: boolean): boolean {
+export function areCookiesAuthorized(useSecureCookie?: boolean): boolean {
```

Comment:
IMO, it means that we can perform this check as unsecure when we don't know if we should be secure or not, which is not great.
what about keeping the same strategy as before (`mustUseSecureCookie(userConfiguration)`) or proposing another strategy that avoid this optional boolean?